moma_controllers/moma_joint_space_controller/src/controller.cpp

```diff
@@ -149,17 +149,15 @@ void JointSpaceController::update(const ros::Time& time, const ros::Duration& pe
 
     read_state();
     bool all_reached = true;
+    std::stringstream ss;
     for (int i = 0; i < n_joints_; i++) {
       double joint_error, velocity_command;
+      
       angles::shortest_angular_distance_with_large_limits(
           q_[i], joint_desired_now(i), lower_limit_[i], upper_limit_[i], joint_error);
-      std::stringstream ss;
-      ss << "Joint current=" << q_[i] << std::endl;
-      ss << "joint desired=" << joint_desired_now(i) << std::endl;
-      ss << "lower limit=" << lower_limit_[i] << std::endl;
-      ss << "upper limit=" << upper_limit_[i] << std::endl;
```

Comment:
Are the lower and upper limits not also relevant for debugging?

Reply:
It is there to check that the current desired value is within limits.